acceptance_tests/features/AddReport.feature

```diff
@@ -1,27 +1,25 @@
 Feature: AddReport
-  Additional reports can be added before data is submitted 
+  Additional reports can be added before data is submitted
 
-	Scenario: The confirmation page provides the ability to add additional reports
+	Scenario: The confirm page provides the ability to add additional reports
     When I load the RTM start page
     And I supply the RTM form with the valid url http://www.example.com
     And I submit the RTM form
-    Then I should see the confirmation page
+    Then I should see the confirm page
     Then I should see the url - http://www.example.com
     And I should see a link to add additional reports
-    Then I should clear the session
-  
+
 	Scenario: The additional reports link leds back to the RTM report form
     When I load the RTM start page
     And I supply the RTM form with the valid url http://www.example.com
     And I submit the RTM form
-    Then I should see the confirmation page
+    Then I should see the confirm page
     Then I should see the url - http://www.example.com
     And I should see a link to add additional reports
     When I click on the additional report link
     Then I should see the RTM form
-    Then I should clear the session
-  
-	Scenario: The confirmation lists multiple reports when more than one report is submitted
+
+	Scenario: The confirm lists multiple reports when more than one report is submitted
```

Comment:
'The confirm page lists ...' - I think this was missing previously anyway but makes more sense